djangocms_content_expiry/cms_config.py

```diff
@@ -1,6 +1,14 @@
 from django.conf import settings
 
-from cms.app_base import CMSAppConfig
+from cms.app_base import CMSAppConfig, CMSAppExtension
+
+
+class ContentExpiryAppExtension(CMSAppExtension):
+    def __init__(self):
+        from .monkeypatch import admin as monkeypatch_admin
+
+    def configure_app(self, cms_config):
```

Comment:
I don't think that we need configure_app here, it looks to be implemented and will be inherited: https://github.com/django-cms/django-cms/blob/release/4.0.x/cms/app_base.py#L114

Reply:
Removed from cms_config